cves/CVE-2010-4704.yml

```diff
@@ -280,8 +314,12 @@ ipc:
     software system reads is another form of IPC.
 
     Answer should be boolean.
-  answer:
-  note:
+  answer: true
+  note: |
+    Because this vulnerability affected the decoding process of a .ogg file,
+    which involves converting digital signals from an .ogg file to analog
+    signals that are humanly audible, we can say that the affected feature
+    used IPCs.
```

Comment:
By this logic, I think just about any vulnerability would be considered IPC in ffmpeg - does the feature work with IPC features directly? (Probably not?)